.github/workflows/arm-auto-signoff.yaml

```diff
@@ -1,20 +1,18 @@
 name: ARM Auto SignOff
 
 on:
-  issue_comment:
-    types:
-      - edited
   # Must run on pull_request_target instead of pull_request, since the latter cannot trigger on
   # labels from bot accounts in fork PRs. pull_request_target is also more similar to the other
-  # triggers "issue_comment" and "workflow_run" -- they are all privileged# and run in the target
-  # branch and repo -- which simplifies implementation.
+  # trigger "workflow_run" -- they are both privileged and run in the target branch and repo --
+  # which simplifies implementation.
   pull_request_target:
     types:
       # Depends on labels, so must re-evaluate whenever a relevant label is manually added or removed.
       - labeled
       - unlabeled
   workflow_run:
-    workflows: ["ARM Incremental TypeSpec"]
+    workflows:
+      ["ARM Incremental TypeSpec", "Swagger Avocado - Set Status", "Swagger LintDiff - Set Status"]
```

Comment:
Trigger on the "Set Status" workflows, since they set the "Commit Status" we care about.